app/mailers/notifications_mailer.rb

```diff
@@ -62,7 +62,7 @@ def weekly_student_summary(project, summary_stats, did_revert_to_pass)
   end
 
   def top_task_desc(tt)
-    "#{tt[:task_definition].abbreviation} - #{tt[:task_definition].name} #{"- which you need to discuss with your tutor" if tt[:status] == :discuss}"
+    "#{tt[:task_definition].abbreviation} - #{tt[:task_definition].name} #{"Due:  "} #{tt[:task_definition].due_date} #{"- which you need to discuss with your tutor" if tt[:status] == :discuss}"
```

Comment:
The task definition is the base due date, not the individual student due date. It will not have any "extensions" applied to it, for example. You may need to adjust the top tasks function in the Project class to add this data.